integration/ed25519/config_test.go

```diff
@@ -0,0 +1,1859 @@
+/*
+Copyright IBM Corp All Rights Reserved.
```

Comment:
I think a **single** integration test that:

- Has peers and orderers in it
- Transacts on a channel
- The channel contains both ECDSA and ed25519 identities and they're both used in the endorsement policy

Is sufficient.

Reply:
That is my intention. This commit was not meant as a final solution yet but a work in progress. The folder integration/ed25519 will be removed.